projects/plugins/crm/src/automation/commons/triggers/class-contact-updated.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Automattic\Jetpack\CRM\Automation\Triggers;
+
+use Automattic\Jetpack\CRM\Automation\Automation_Exception;
+use Automattic\Jetpack\CRM\Automation\Automation_Workflow;
+use Automattic\Jetpack\CRM\Automation\Base_Trigger;
+
+class Contact_Updated extends Base_Trigger {
+	
+	private $contact_before_update = array();
+	
+	public function __construct() {
+		$trigger_data = array(
+			'name'        => 'contact_updated',
+			'title'       => 'Contact Updated',
+			'description' => 'Triggered when a CRM contact is updated',
```

Comment:
These looks like they could benefit from translation 😄

```suggestion
			'title'       => __( 'Contact Updated', 'zero-bs-crm' ),
			'description' => __( 'Triggered when a CRM contact is updated', 'zero-bs-crm' ),
```